dts/bindings/clock/nxp,kinetis-mcg.yaml

```diff
@@ -14,5 +14,19 @@ properties:
   "#clock-cells":
     const: 1
 
+  fcrdiv:
+    type: int
+    description: |
+      Internal Reference Clock Divider.
+      Division factor is given as 2^fcrdiv.
+      The valid range is 0 to 7.
+
+  lircdiv2:
+    type: int
+    description: |
+      Second Low-frequency Internal Reference Clock Divider.
+      Division factor is given as 2^lircdiv2.
+      The valid range is 0 to 7.
```

Comment:
why can't these be enums?

Reply:
@nordicjm, thank you for review. Changed to enum.